doc/manual/faq.rst

```diff
@@ -125,6 +125,39 @@ This and other similar messages almost always indicate that your device database
 
 to let the controller manager start the necessary controllers automatically.
 
+fix ``address already in use`` when running ARTIQ commands?
+-----------------------------------------------------------
+
+A message like ``OSError: [Errno 98] error while attempting to bind on address ('127.0.0.1', 1067): [errno 98] address already in use`` indicates that the IP address and port number combination you're trying to use is already occupied by some other process. Often this simply means that the ARTIQ process you're trying to start is in fact already running. Note for example that trying to start a controller which is already being run by a controller manager will generally fail for this reason.
+
+.. note::
+    ARTIQ management system communications, whether distributed or local, run over TCP/IP, using TCP port numbers to identify their destinations. Generally speaking, client processes like the dashboard don't require fixed ports of their own, since they can simply reach out to the master when they want to establish a connection. Running multiple dashboards will never cause a port conflict. On the other hand, server processes like the ARTIQ master have to be 'listening' at a fixed, open port in order to be able to receive incoming connections. For more details, look into `ports in computer networking <https://en.wikipedia.org/wiki/Port_(computer_networking)>`_.
+
+    Most management system processes belong to the second category, and are bound to one or several fixed communication ports while they're running. See also :doc:`default_network_ports`.
+
+You can use the command ``netstat`` to list the ports currently in use on your system. To check the status of a specific port on Linux, try either of: ::
+
+    $ netstat -anp --inet | grep "<port-number>"
+    $ lsof -i:<port-number>
+
+On Windows, use ::
+
+    $ netstat -ano -p ip | find "<port-number>"
```

Comment:
This command is not exactly correct:
In windows cmd: `netstat -ano -p TCP | find "1383"`
In windows powershell: `netstat -ano -p TCP | Select-String "1383"`

But actually the MSYS2 also can call some windows commands: `netstat -ano -p TCP | grep "1383"`

Reply:
Only netstat works same across shells, find command is different on every platform, so I guess it is better to say explicitly for each, since conda and msys2 are up to three different shells.
~~Or observe by eyes~~